macros/Compat.ejs

```diff
@@ -37,7 +37,7 @@ var output = '';
 var category = query.split(".")[0];
 var legendItems = new Set(); // entries will be unique
 
-var compatStrings = string.deserialize(template('L10n:CompatTable'));
+var compatStrings = string.deserialize(await template('L10n:CompatTable'));
```

Comment:
@davidflanagan Is there a reason we still have some uses of `var` even though in other places they are changed to `let` or `const`?

Reply:
I tried to keep the changes to the macros/ directory minimal. It was not my goal to modernize the JavaScript in the macros.

Obviously, I had to add await (and some asyncs) to the macros. There were also a few cases where I had to change from var to let to get the macros to work correctly.  IIRC the main issue was where we had `var page`. That declaration shadows the page.* API. And for some reason related to moving the DekiScript-page.ejs macro into src/environment.js I needed to change all `var page` declarations into `let page`.  

And I think that sometimes when I was changing one var to a let, I'd go ahead and change the surrounding ones as well.  But there was no intent to do it everywhere (because that can alter the behavior of code, and I didn't want to risk it.)